From 76264686f00fbe7f452b55e14ff1fa3df2bda2b8 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Sat, 29 Jan 2022 01:03:34 +0200
Subject: [PATCH] [SW-65638] gaudi2: release: fix set recovery

If no explicitly requested the recovery mode should not be activated.

Change-Id: I227f29051566d25fffbf51aac5472c471d326a05
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/scripts/fwi/hls_platform.rc b/scripts/fwi/hls_platform.rc
index 18272dd..36c0a71 100644
--- a/scripts/fwi/hls_platform.rc
+++ b/scripts/fwi/hls_platform.rc
@@ -769,8 +769,9 @@
 function oams_set_recovery_mode_op()
 {
 	local mode
+
 	case ${recovery_args[0]:-"NONE"} in
-	NONE) ;;
+	NONE) return 0;;
 	off|OFF|dis|disable) mode=disable ;;
 	on|ON|enb|enable) mode=enable ;;
 	*)
